### PR TITLE
fix(ci): image builder

### PR DESCRIPTION
**Pull request by c3ppc3pp**
This PR resolves some discrepancy around building the "builder image"

This PR resolves the chicken-egg issue in the pipeline by removing the builder image, but using a "bare" VM to install the few stuff we need and build/push/sign the image.

### CODE REVIEW

**Review by polaroi8d — APPROVED** [review]
Looks good to me, but please wait for @nandor-magyar's approval.

**Review by nandor-magyar — APPROVED** [review]
LGTM

**Pull request merged**
